mobile/src/bundles/talent/components/skills-and-projects-form/styles.ts

```diff
@@ -0,0 +1,28 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+    container: {
+        backgroundColor: '#FFFFFF',
+        flexGrow: 1,
+    },
+    notConsiderContainer: {
+        gap: 5,
+        flexWrap: 'wrap',
+        width: 200,
+    },
+    tagContainer: {
+        gap: 5,
+        flexWrap: 'wrap',
+    },
+    links: {
+        gap: 5,
+        position: 'relative',
+    },
+    linksBtn: {
+        position: 'absolute',
+        top: 5,
+        right: 5,
+    },
```

Comment:
Because i need to place delete button and I decided to place it in the right corner of the input
![projectLinks](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/104503014/871076dc-0d3a-4074-bd5d-24c5607a78e7)